.github/workflows/update-dev-website.yml

```diff
@@ -0,0 +1,60 @@
+name: Update dev website
+on:
+  [push, pull_request]
```

Comment:
The workflow must be restricted to **develop** branch only (similar like the release workflow is restricted to the main branch)